content/docs/release-notes/2305-release-notes.md

```diff
@@ -0,0 +1,191 @@
+* Animation Asset Import Improvements ([#12387](https://github.com/o3de/o3de/issues/12387))
+	* A number of quality of life improvements to the animation import process to make the overall experience more robust and straightforward.
+* Animation Editor Unified Inspector Window ([#10666](https://github.com/o3de/o3de/issues/10666))
+	*  A significant overhaul was made to the O3DE Animation Editor UI to bring it more inline with other O3DE tools. Numerous panels were either combined or removed to improve usability and consistency.
+* Animation Editor AnimGraph Performance Visualizer ([#13490](https://github.com/o3de/o3de/pull/13490))
+	* Added an option to display timing information for nodes in an AnimGraph to give real-time information on how long each node is taking to process. This gives invaluable information when profiling/optimizing an AnimGraph. _A big thank to our friends at FragLab for contributing this feature back to O3DE_.
+* Character Performance Improvements ([#13920](https://github.com/o3de/o3de/pull/13920), [#14461](https://github.com/o3de/o3de/pull/14461))
+	* Improved performance in several character systems.
+* Physics Profiling and Performance Improvements ([#13169](https://github.com/o3de/o3de/pull/13169), [#13258](https://github.com/o3de/o3de/pull/13258), [#13061](https://github.com/o3de/o3de/pull/13061), [#13021](https://github.com/o3de/o3de/pull/13021), [#14666](https://github.com/o3de/o3de/pull/14666), [#14350](https://github.com/o3de/o3de/pull/14350))
+* Physics Viewport Improvements ([#14199](https://github.com/o3de/o3de/pull/14199))
+	* Physics colliders can now be clicked and selected in the 3D viewport
+* Add support for reduced coordinate articulations ([#14851](https://github.com/o3de/o3de/issues/14851) - PhysX 5.1 only)
+	* Reduced coordinate articulations are incredibly important for both robust and accurate simulation of joints. They are used extensively in robotics simulations when simulating devices such as a robotic arm.
+	* This feature is in an early experimental state and must be enabled in the Settings Registry option: `/Amazon/Physics/EnableReducedCoordinateArticulations`
+* PhysX simulation runs on multiple threads on Linux. ([#14075](https://github.com/o3de/o3de/pull/14075))
+* Removed deprecated Physics and Blast legacy materials. ([#9840](https://github.com/o3de/o3de/issues/9840), [#9839](https://github.com/o3de/o3de/issues/9839))
+* Blast Gem moved to experimental branch in o3de-extras. ([#13584](https://github.com/o3de/o3de/pull/13584))
+
+## sig-testing
+
+* Feature 1
+* Feature 2
+
+## sig-ui-ux
+* Feature 1
+* Feature 2
+
+## Known Issues
+
+* Levels and prefabs containing Polygon Prism components need to be re-saved when switching from PhysX 4 to PhysX 5.1 and vice versa.
+
+## New and Noteworthy
+
+* PhysX 5.1 shows a 15% increase in simulation performance compared with PhysX 4.
+* With PhysX 5.1, mobile assets now use the more optimal structure eBVH34.
+
```

Comment:
Please pull in the recent changes from https://github.com/o3de/sig-release/commit/44eb938226491a1a42e4f1354d2cf0ec938e99f0